workflow/scripts/collect-fp-fn-benchmarks.py

```
@@ -0,0 +1,42 @@
+    if col.name == "callset":
+        return col
+    if col.name == "coverage":
+        return col.apply(cov_key)
+    else:
+        return col
+
+
+results.sort_values(["callset", "coverage"], inplace=True, key=sort_key)
+results["sort_index"] = results["coverage"].apply(cov_key)
+
+results.to_csv(snakemake.output[0], sep="\t", index=False)
```

Ruff (0.8.0)

Undefined name `snakemake`

(F821)